CMakeLists.txt

```diff
@@ -563,6 +580,9 @@ if(WIN32 OR WINCE)
    SET(CMAKE_RC_COMPILE_OBJECT "<CMAKE_RC_COMPILER> -O coff -I ${CMAKE_CURRENT_SOURCE_DIR}/navit/gui/win32/resources <DEFINES> -i <SOURCE> -o <OBJECT>")
    set_with_reason(support/ezxml "Windows detected" TRUE)
    set_with_reason(speech/espeak "Windows detected" TRUE)
+   #speech/espeak does use windows threads, and therefore needs only parts of
+   #espeak lib. INTERNAL_ESPEAK controls this.
+   set_with_reason(INTERNAL_ESPEAK "Windows detected" FALSE)
```

Comment:
No this is OK. INTERNAL_ESPEAK controls if support/espeak builds the complete libespeak or just parts of it. And the (old) Windows version does only want parts of it, so this is set to FALSE.
Think I'll rename INTERNAL_ESPEAK to better reflect this :)